Core/TestsFoundation/Helpers/OfflineHelper.swift

```diff
@@ -0,0 +1,34 @@
+//
+// This file is part of Canvas.
+// Copyright (C) 2024-present  Instructure, Inc.
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as
+// published by the Free Software Foundation, either version 3 of the
+// License, or (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+//
+
+public class OfflineHelper: BaseHelper {
```

Comment:
There's already an "OfflineHelper", it's just not a separated helper class. You can find it in DashboardHelper.Options.OfflineContent.
I think we should stick to only one of these. So either keep this OfflineHelper and then update all of the other offline tests to use this one, or update the other one with these variables and functions and delete this one. I think the faster solution is to let go of this and use the other one, because otherwise you have to move everything from the other one to here and update every single test case accordingly. But I'll leave this up to you now. The main point is that now we have 2 helper for the same thing and it's neither effective nor clear in the test cases (now there's "Offline" and "OfflineHelper" too).

Reply:
I put these page object nodes to the DashboardHelper.Options.OfflineContent yet, but I think later we should refactor it in a separate ticket, because that's a bit confusing to have all the Offline related nodes within the "DashboardHelper" since for example this 'Offline Mode' no internet connection alert has nothing to deal with the Dashboard. But I don't want to change anything drastically by my own, so we may can discuss it laters if we would like to refactor this in a separate ticket. Yet, I'm just learning and adopting to existing things and conventions, so I put my node elements to this existing struct. :)